shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2398,12 +2398,11 @@ int WebContents::GetProcessID() const {
 }
 
 base::ProcessId WebContents::GetOSProcessID() const {
-  base::ProcessHandle process_handle = web_contents()
-                                           ->GetPrimaryMainFrame()
-                                           ->GetProcess()
-                                           ->GetProcess()
-                                           .Handle();
-  return base::GetProcId(process_handle);
+  return web_contents()
+      ->GetPrimaryMainFrame()
+      ->GetProcess()
+      ->GetProcess()
+      .Pid();
```

Comment:
`Pid()` does a validity check https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:base/process/process_posix.cc;l=305 which will not be true synchronously when creating a child process. Given we don't provide a process spawn event for renderer process similar to https://github.com/electron/electron/blob/main/docs/api/utility-process.md#event-spawn we cannot reliably check in this line https://github.com/electron/electron/blob/13f9e2db40df5537c13ee72a92fad13a0c09b730/spec/api-web-contents-spec.ts#L921 , I would suggest it should be removed.